tests/test_resolutions.py

```diff
@@ -123,7 +123,6 @@ def test_can_be_instantiated(self) -> None:
 
     def test_the_test(self) -> None:
         cim = ContainsItemMatching(r"([Ss]pam ?)+").resolve()
-
```

Comment:
Please put this back in. It separates the Arrange from the Assert. :P

Reply:
fixed